src/dynamicTypes/typesLoader.ts

```diff
@@ -18,9 +16,20 @@ type FullCorvidTypes = {
   libs: Array<Lib>;
 };
 
-export const fullCorvidTypes = fullCorvidTypesJSON as FullCorvidTypes;
+const dynamicallyImportFullCorvidTypes = async (): Promise<{
+  default: FullCorvidTypes;
+}> => {
+  return import(
+    /* webpackChunkName: "fullCorvidTypesJSON" */ "../../dist/fullCorvidTypes.json"
+  );
+};
+
+export const getCollectionLibs = async (
+  key: LibCollections
+): Promise<Lib[]> => {
+  const fullCorvidTypesModule = await dynamicallyImportFullCorvidTypes();
```

Comment:
tagid, this code doesn't mean that every time we are changing context we are importing the chunk again?
i think we should import it once

Reply:
No, webpack is smart to load it from the network only once.
Just checked with:

```
  console.time("corvid-types-import");
  const fullCorvidTypesModule = await dynamicallyImportFullCorvidTypes();
  console.timeEnd("corvid-types-import");
```

This results in similar times if I only call import once and store the value for caching or calling import every time.